src/renderer/components/Bonds/Bonds.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import { withKnobs, select } from '@storybook/addon-knobs'
+import { useCallback, useState } from '@storybook/addons'
+import { Meta, Story } from '@storybook/react'
+import { baseAmount } from '@xchainjs/xchain-util'
+
+import { getMockRDValueFactory, RDStatus } from '../../../shared/mock/rdByStatus'
+import { mockValidatePassword$ } from '../../../shared/mock/wallet'
+import { AddressValidation } from '../../services/thorchain/types'
+import { ApiError, ErrorId } from '../../services/wallet/types'
+import { Bonds } from './Bonds'
+import { NodeInfo } from './types'
+
+const getMockRDValue = getMockRDValueFactory<ApiError, NodeInfo>(
+  () => ({
+    bond: baseAmount(100000000 * 40000000),
+    award: baseAmount(100000000 * 400000),
+    status: 'active'
+  }),
+  () => ({
+    msg: 'error message',
+    errorId: ErrorId.GET_NODE_INFO
+  })
+)
+
+const addressValidation: AddressValidation = (_) => true
+
+export const Default: Story = () => {
+  const [nodesList, setNodesList] = useState<string[]>([])
+
+  const nodesSelect: Record<string, RDStatus> = nodesList
+    .map((node) =>
+      select(
+        node,
+        {
+          initial: 'initial',
+          pending: 'pending',
+          error: 'error',
+          success: 'success'
+        },
+        'pending'
+      )
+    )
+    .reduce((acc, node, index) => ({ ...acc, [index]: node }), {})
+
+  const removeNode = useCallback(
+    (node: string) => {
+      setNodesList(nodesList.filter((current) => current !== node))
+    },
+    [nodesList, setNodesList]
+  )
+  const addNode = useCallback((node: string) => setNodesList([...nodesList, node]), [nodesList, setNodesList])
+
+  return (
+    <Bonds
+      addressValidation={addressValidation}
+      network={'testnet'}
+      addNode={addNode}
+      validatePassword$={mockValidatePassword$}
+      removeNode={removeNode}
+      goToNode={(node) => console.log('go to ', node)}
+      nodes={nodesList.map((node, index) => ({
+        nodeAddress: node,
+        data: getMockRDValue(nodesSelect[index])
+      }))}
+    />
+  )
+}
+Default.storyName = 'default'
+
+const meta: Meta = {
+  component: Bonds,
+  title: 'Bonds',
+  decorators: [withKnobs]
```

Comment:
As discussed: Pls use [Storybook Controls](https://github.com/storybookjs/storybook/tree/master/addons/controls#how-will-this-replace-addon-knobs) for any new story (knobs will be [deprecated in the future](https://github.com/storybookjs/storybook/tree/master/addons/controls#how-will-this-replace-addon-knobs))

Reply:
in a conclusion as discussed in DM:
For now  there is no way to make controls dynamic yet: 
 [Unfortunately controls does not support anything equivalent to dynamic knobs.](https://github.com/storybookjs/storybook/issues/11984#issuecomment-673537924)

so we decided to use knobs for now